### ISSUE TITLE
File manager error - Uncaught TypeError: Cannot call method 'setValue' of undefined 

### ISSUE THREAD

**Issue by MrKarlDilkington** [ExchangeCore/Concrete5-CKEditor#25]
Steps to recreate error:
- click Image plugin button
- click Browse Server button
- select an image

```
Uncaught TypeError: Cannot call method 'setValue' of undefined     plugin.js
```

Commenting out these three lines in plugin.js allows the file manager to work without the error.
https://github.com/ExchangeCore/Concrete5-CKEditor/blob/develop/assets/concrete5filemanager/plugin.js#L27-L29

The img to picture element replacement is working well.

**Comment by joe-meyer** [comment]
The problem occurs when the Enhanced Image plugin is enabled. Looks like they don't use the same field names.

**Comment by joe-meyer** [comment]
Fixed in a96798670cddfc8c766d1283d3a61072655ea5c6
